### DISCUSSION TITLE
Will it work with Raspberry Pi Zero W ???

### DISCUSSION THREAD

**Discussion by rodjavier** [frntc/RAD#47] [Q&A]
Hi everybody. I have a Pi Zero W v1.1 from 2017 and wondering if this can work for RAD instead of the Pi Zero 2 ??? any comments/ideas welcome. Thanks a lot!

**Comment by frntc** [comment]
No, the Pi Zero 1.1 is a completely different SoC, it won't work (not even boot the firmware)

**Reply by rodjavier** [reply]
Hi again! Thanks for your answer. I have now the Pi Zero 2 from 2021 and got the RAD module, installed, follow all the instructions but the question is: Is this compatible with the C64 Reloaded MK2 (latest firmware from November 2023) board?
I am waiting to see if a friend of mine can stop by with his old C64 breadbin and try the RAD module, but so far i get no responses on the reloaded mk2 board. Any knowledge from your side about this topic? Thanks again!!!

**Reply by frntc** [reply]
Hi, I don't have a Reloaded board myself, but I think some people reported that it worked. What happens if you turn it on? (black screen?)

**Reply by rodjavier** [reply]
When i turn it on it goes directly to the Basic commodore screen. If i press the reset button it does a strange quick flickering and if i press the menu button it does nothing. I have the epyx fast loader cartridge and works fine, have the midi cartridge also fine but this RAD one does nothing. Pi board seems ok cause i have power and video output. Is there any tweak on the config file worth trying? Or any tests that can be perform on the Pi?
